WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductVariationSelectorViewModel.swift

```diff
@@ -16,6 +16,11 @@ final class ProductVariationSelectorViewModel: ObservableObject {
     ///
     private let stores: StoresManager
 
+    /// Determines if multiple item selection is supported, `false` by default.
+    /// Mainly used to allow multi-selection on the Order Creation flow
```

Comment:
We should not include this comment as this is too specific for a shared view, which makes it sound like it's not applicable to other features 😅

Reply:
Removed! https://github.com/woocommerce/woocommerce-ios/pull/8935/commits/a52c1b23598c20a0ebe6fba77b651cc984b4b916